countdown.rb

```diff
@@ -5,64 +5,115 @@
 
 require_relative 'helpers'
 
-clear
 
+a = Artii::Base.new :font => 'ogre'
+b = Artii::Base.new :font => 'catwalk'
+c = Artii::Base.new :font => 'cosmic'
+d = Artii::Base.new :font => 'invita'
+clear
 puts 'Introduce el tiempo en minutos'
 minutes = gets.chomp.to_i
 
 clear
 
 puts 'Introduce la clave de desactivación'
 bomb_key = gets.chomp
-
+accumulated_key = []
+(bomb_key.length).times { accumulated_key << "*" }
 bomb_deactivated = false
 
 final = Time.now + minutes * 60
 
 loop do
-  accumulated_key = []
+  
   pressed_key = nil  
+  count = 0
   begin
     clear
-    puts "SE ESTÁ ACABANDO EL TIEMPO..."
-    puts "#{format_time(time_remaining(final))}"
-
-    puts "Introduce la clave:\n\n"
-
-    print "#{accumulated_key.join}"
+    3.times { puts }
```

Comment:
It would be handy to have a function like `spaced_print(text)` that adds the 3 newlines before printing the sentence.